railties/lib/rails/generators/rails/app/templates/config/cable.yml

```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
I took as guide the other templates, @palkan, see the following: [postgresql.yml](https://github.com/MrOutis/rails/blob/e267db2914785e20bee17097247dc84539ea4c39/railties/lib/rails/generators/rails/app/templates/config/databases/postgresql.yml)

Honestly, I don't know how it works, but I think is kind of "escaping" the ERB interpolation.